action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Jira Check Issue Status'
+description: 'check the Jira issue status'
+author: 'Michael Hakim'
+inputs:
+  organization-name: 
+    description: 'The name of the organization on Jira'
+    required: true
+    default: 'gianteagle'
+  jira-token:  
+    description: 'Jira Token'
+    required: true
+  jira-user:
+    description: 'Jira User'
+    required: false
+runs:
+  using: "composite"
+  steps:
+      - name: First Step
+        shell: bash
+        run: |
+          echo "${GITHUB_HEAD_REF}"
+          grep -Eo "[aA-zZ]+-[0-9]+" | tr '[:lower:]' '[:upper:]'
+      - name: Extract Issue name
+        shell: bash
+        run: |
+          if [[ "$(echo ${GITHUB_HEAD_REF} | cut -d "/" -f1)" =~ (feature|hotfix|bugfix) && ! "$(echo ${GITHUB_HEAD_REF} | cut -d "/" -f2)" =~ [A-Z]+-[0-9]+ ]]; then
+            echo "Your PR is not connected to an issue on Jira, please create an issue for it."
+            exit 1
+          else
+            echo "##[set-output name=name;]$(echo ${GITHUB_HEAD_REF} | grep -Eo '[aA-zZ]+-[0-9]+' | tr '[:lower:]' '[:upper:]')"
+          fi 
+        id: issue_name
+      - name: Test
```

Comment:
Just for debug?

Reply:
Actually I can keep that but I will change the task name